bin/evm2wasm.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env nodejs
```

Comment:
Why is this needed?

Reply:
This one was a leftover from when @lrettig and I tried the Javascript translator, Node is installed as nodejs in Debian.

Reverted in https://github.com/ewasm/evm2wasm/pull/201/commits/e5199247e992d81c75f98c88650a9b537f71bf48